submissions/AndriiZahumnyi/js-dom/index.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Document Object Model - Practical task</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="css/main_style.css">
+</head>
+
+<body>
+  <div class="container">
+    <header class="header">
+      <h1 class="logo-wide">Document Object Model - Practical task</h1>
+    </header>
+    <main class="main">
+      <aside class="aside-container">
+          <div class="add-container">
+            <div class="logo-wrapper">
+              <h1 class="logo-thin">DOM - Practical task</h1>
+            </div>
+            <div class="toggle-div"> 
+              <input type="checkbox" id="menu-toggle" class="menu-toggle" />
```

Comment:
it's better to make this element as a button. It will be more correct semantically